dev-python/colorlog/colorlog-3.1.4.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} pypy pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Log formatting with colors"
+HOMEPAGE="https://pypi.org/project/colorlog/ https://github.com/borntyping/python-colorlog"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND=""
+DEPEND="${RDEPEND}
```

Comment:
`DEPEND=${RDEPEND}` is usually wrong for Python packages which don't link with their runtime dependencies. It's even weirder with an empty RDEPEND.